src/components/App.js

```diff
@@ -23,11 +24,44 @@ class App extends React.Component {
     return (
       <Provider store={store}>
         <I18nextProvider i18n={i18n}>
-          <Router
-            history={browserHistory}
-            children={routes}
-            render={applyRouterMiddleware(useScroll())}
-          />
+          <div>
+            <Helmet defaultTitle='inspire.data.gouv.fr'>
```

Comment:
Also, you can even have sub title templates, like on twitter.com

For example on a search page:

```js
<Helmet
  titleTemplate='%s | Twitter Search'
  defaultTitle='Twitter Search'
  title={query.q}
/>
```